doc/ExpertOptions.md

```diff
@@ -1475,28 +1475,29 @@ You would write the following manifest:
 }
 ```
 
-When you compile the workflow, provide the manifest using the `-inputs` option, and it will be translated to:
-
-`mymanifest.dx.json`
```

Comment:
How about leaving the title of the workflow above the snippet? Similarly to other snippets in this area of the doc.

Reply:
fixed in [34af2d0](https://github.com/dnanexus/dxCompiler/pull/354/commits/34af2d06912bb083b83ecf159a95c62e9e91a92f)[34af2d0](https://github.com/dnanexus/dxCompiler/pull/354/commits/34af2d06912bb083b83ecf159a95c62e9e91a92f)